README.md

```diff
@@ -5,48 +5,80 @@
+distribution over the simulator’s parameters based on observed data. This distribution
+indicates the most likely parameter values while additionally quantifying uncertainty
+and revealing potential interactions between parameters.
 
-`sbi` offers a simple interface for posterior inference in a few lines of code
+### Key Features of `sbi`
+
+`sbi` offers a blend of flexibility and ease of use:
+
+- **Low-Level Interfaces**: For those who require maximum control over the inference
+  process, `sbi` provides low-level interfaces that allow you to fine-tune many aspects
+  of your work.
+- **High-Level Interfaces**: If you prefer simplicity and efficiency, `sbi` also offers
+  high-level interfaces that enable quick and easy implementation of complex inference
+  tasks.
+
+In addition, `sbi` supports a wide range of state-of-the-art inference algorithms (see
+below for a list of implemented methods):
+
+- **Amortized Methods**: These methods enable the reuse of posterior estimators across
+  multiple observations without the need to retrain.
+- **Sequential Methods**: These methods focus on individual observations, optimizing the
+  number of simulations required.
+
+Beyond inference, `sbi` also provides:
+
+- **Validation Tools**: Built-in methods to validate and verify the accuracy of your
+  inferred posteriors.
+- **Plotting and Analysis Tools**: Comprehensive functions for visualizing and analyzing
+  results, helping you interpret the posterior distributions with ease.
+
+Getting started with `sbi` is straightforward, requiring only a few lines of code:
 
 ```python
 from sbi.inference import SNPE
-# import your simulator, define your prior over the parameters
-# sample parameters theta and observations x
+# Given: parameters theta and corresponding simulations x
 inference = SNPE(prior=prior)
-_ = inference.append_simulations(theta, x).train()
+inference.append_simulations(theta, x).train()
 posterior = inference.build_posterior()
 ```
 
-## Installation
+### Installation
 
-`sbi` requires Python 3.8 or higher. A GPU is not required, but can lead to speed-up in some cases. We recommend to use a [`conda`](https://docs.conda.io/en/latest/miniconda.html) virtual
-environment ([Miniconda installation instructions](https://docs.conda.io/en/latest/miniconda.html)). If `conda` is installed on the system, an environment for installing `sbi` can be created as follows:
+`sbi` requires Python 3.9 or higher. While a GPU isn't necessary, it can improve
+performance in some cases. We recommend using a virtual environment with
+[`conda`](https://docs.conda.io/en/latest/miniconda.html) for an easy setup.
 
-```commandline
-# Create an environment for sbi (indicate Python 3.8 or higher); activate it
-$ conda create -n sbi_env python=3.10 && conda activate sbi_env
-```
+To install `sbi`, follow these steps:
 
-Independent of whether you are using `conda` or not, `sbi` can be installed using `pip`:
+1. **Create a Conda Environment** (if using Conda):
```

Comment:
do we really have to explain that people should create a conda env?

Reply:
Yes, I think that's good for some users.